.github/workflows/msys2.yml

```diff
@@ -32,8 +32,8 @@ jobs:
           MINGW_INSTALLS: ${{ matrix.msystem }}
           PKGEXT: ".pkg.tar.xz"
         shell: msys2 {0}
-      - name: "Upload binaries"
-        uses: actions/upload-artifact@v2
-        with:
-          name: mingw-w64-${{ matrix.arch }}-tiledb
-          path: .github/workflows/mingw-w64-tiledb/*.pkg.tar.*
+      #- name: "Upload binaries"
```

Comment:
Sorry, but I try to stick to the "no dead code" rule. Can we delete this? We can always dig it up from git history later.

Reply:
Done in a8d253fd (following a rebase).